src/components/tabs/README.md

```diff
@@ -26,7 +26,7 @@ attribute, see https://vuejs.org/v2/api/#key).
 
 ## Cards integration
 
-Tabs support integrating with Bootstrap cards. Just add the `card` property to `<b-tabs>`. and place
```

Comment:
I think this was meant to be a comma... maybe...

Reply:
But I think it is ok without any punctuation.

Reply:
As far as I know in English it is unusual to use comma before "and".

Reply:
"it is possible, _and_ usually unusual"

:laughing: